test/run-shared-integration-tests

```diff
@@ -78,7 +78,7 @@ run-test-for-cluster-size() {
     # Run the tests and buffer the output to a log file. We'll display it later
     # if the test fails. This avoids interleaving of output to the terminal
     # when tests are running in parallel.
-    node "${ringpop_common_dir}/test/it-tests.js" \
+    node "${ringpop_common_dir}/test/it-tests.js" --enable-feature partition-healing \
```

Comment:
nit: for easier to understandable diffs now and in the future I think it is easiest if we put `--enable-feature` flags here on separate lines. It will cause only 1 addition instead of a remove and an addition when adding tests. Same when we can remove features because they became mandatory.